codec/grpc/grpc_codec.go

```diff
@@ -25,6 +31,12 @@ func (Codec) Unmarshal(data []byte, v interface{}) error {
 	if !ok {
 		return fmt.Errorf("failed to unmarshal, message is %T (missing vtprotobuf helpers)", v)
 	}
+	//All types that implement github.com/golang/protobuf/proto.Message have a Reset method
+	vv, ok := v.(reseter)
```

Comment:
I can't require the message to implement `ResetVT()` because this method isn't generated by default (it requires the `pool` option)
Doing so would break the codec for user who just use the `marshal` and `umarshal` features

But I can follow this style and do 
```suggestion
	vv, ok := v.(interface{Reset()})
```

Reply:
You're right :) IMO the resetvt func should be turned into a plug-in that's activated automatically with pool plugin.

Reply:
Don't you mean that `ResetVT()` should be generated automatically when the unmarshal plugin is enabled ?